app/models/lesson_preview.rb

```diff
@@ -0,0 +1,3 @@
+class LessonPreview < ApplicationRecord
+  validates :content, presence: true, length: { maximum: 70_000 }
```

Comment:
Nit: To validate this length on the client side, can we add a `maxLength={10}` attribute to the text field [here](https://github.com/TheOdinProject/theodinproject/blob/main/app/javascript/components/lesson-preview/components/lesson-content-input.jsx#L10) please?